include/z64cutscene_commands.h

```diff
@@ -11,11 +11,571 @@
+ * FORMAT
+ *   Capital U is Unused
+ *   mmmmssss eeeeUUUU
+ *   size = 0x08
+ */
+#define CS_SCENE_UNK_132(base, startFrame, endFrame) \
+    { CMD_HH(base, startFrame) }, { CMD_HH(endFrame, endFrame) }
+
+
+/**
+ * ARGS
+ *   s32 entries (e)
+ * FORMAT
+ *   00000133 eeeeeeee
+ *   size = 0x8
+ */
+#define CS_STOPSEQ_LIST(entries) { CS_CMD_STOPSEQ }, { CMD_W(entries) }
+
+/**
+ * ARGS
+ *   s16 base (m), s16 startFrame (s), s16 endFrame (e)
+ * FORMAT
+ *   Capital U is Unused
+ *   mmmmssss eeeeUUUU
+ *   size = 0x08
+ */
+#define CS_STOPSEQ(base, startFrame, endFrame, unk_06) \
+    { CMD_HH(base, startFrame) }, { CMD_HH(endFrame, unk_06) }
+
+
+/**
+ * ARGS
+ *   s32 entries (e)
+ * FORMAT
+ *   00000134 eeeeeeee
+ *   size = 0x8
+ */
+#define CS_PLAYAMBIENCE_LIST(entries) { CS_CMD_PLAYAMBIENCE }, { CMD_W(entries) }
+
+/**
+ * ARGS
+ *   s16 base (m), s16 startFrame (s), s16 endFrame (e)
+ * FORMAT
+ *   Capital U is Unused
+ *   mmmmssss eeeeUUUU
+ *   size = 0x08
+ */
+#define CS_PLAYAMBIENCE(base, startFrame, endFrame, unk_06) \
+    { CMD_HH(base, startFrame) }, { CMD_HH(endFrame, unk_06) }
+
+
+/**
+ * ARGS
+ *   s32 entries (e)
+ * FORMAT
+ *   00000135 eeeeeeee
+ *   size = 0x8
+ */
+#define CS_FADEAMBIENCE_LIST(entries) { CS_CMD_FADEAMBIENCE }, { CMD_W(entries) }
+
+/**
+ * ARGS
+ *   s16 base (m), s16 startFrame (s), s16 endFrame (e)
+ * FORMAT
+ *   Capital U is Unused
+ *   mmmmssss eeeeUUUU
+ *   size = 0x08
+ */
+#define CS_FADEAMBIENCE(base, startFrame, endFrame, unk_06) \
+    { CMD_HH(base, startFrame) }, { CMD_HH(endFrame, unk_06) }
+
+
+/**
+ * ARGS
+ *   s32 entries (e)
+ * FORMAT
+ *   0000015E eeeeeeee
+ *   size = 0x8
+ */
+#define CS_TERMINATOR_LIST(entries) { CS_CMD_TERMINATOR }, { CMD_W(entries) }
+
+/**
+ * ARGS
+ *   s16 base (m), s16 startFrame (s), s16 endFrame (e)
+ * FORMAT
+ *   Capital U is Unused
+ *   mmmmssss eeeeUUUU
+ *   size = 0x08
+ */
+#define CS_TERMINATOR(base, startFrame, endFrame) \
+    { CMD_HH(base, startFrame) }, { CMD_HH(endFrame, endFrame) }
+
+
+/**
+ * ARGS
+ *   s32 entries (e)
+ * FORMAT
+ *   0000015F eeeeeeee
+ *   size = 0x8
+ */
+#define CS_SCENE_UNK_15F_LIST(entries) { CS_CMD_15F }, { CMD_W(entries) }
+
+/**
+ * ARGS
+ *   s16 base (m), s16 startFrame (s), s16 endFrame (e)
+ * FORMAT
+ *   Capital U is Unused
+ *   mmmmssss eeeeUUUU
+ *   size = 0x08
+ */
+#define CS_SCENE_UNK_15F(base, startFrame, endFrame) \
+    { CMD_HH(base, startFrame) }, { CMD_HH(endFrame, endFrame) }
+
+
+/**
+ * ARGS
+ *   s32 entries (e)
+ * FORMAT
+ *   00000004 eeeeeeee
+ *   size = 0x8
+ */
+#define CS_RUMBLE_LIST(entries) { CS_CMD_RUMBLE }, { CMD_W(entries) }
+
+/**
+ * ARGS
+ *   s16 base (m), s16 startFrame (s), s16 endFrame (e)
+ * FORMAT
+ *   Capital U is Unused
+ *   mmmmssss eeeeUUUU
+ *   size = 0x08
+ */
+#define CS_RUMBLE(base, startFrame, endFrame, unk6, unk7, unk8) \
+    { CMD_HH(base, startFrame) }, { CMD_HBB(endFrame, unk6, unk7) }, { CMD_BBBB(unk8, 0, 0, 0) }
```

Comment:
Maybe I'm missing it, but what do all these letters mean in the commands `CMD_HH`, `CMD_HBB`, `CMD_BBBB`, `CMD_W`, is it suppose to be `word/half/byte`? Is that documented somewhere? Is this unique to the cutscene commands or is this a generic thing?